inst/app/server.R

```diff
@@ -16,12 +16,42 @@ server <- function(input, output, session) {
   
   # ----- REACTIVES ----- #
   
+  # filter data according to user selected region
+  react_region_dropdown <- reactive(
+    x = {
+        if(input$input_region == "全部 / All"){
+          
+          data_master_raw
+          
+        } else {
+          
+          
+          data_select <- dplyr::filter(.data = data_master_raw,
+                                       District == input$input_region)
+          
+          return(data_select)
+        }
+    }
+  )
+  
+  # renderUI for second dropdown for constituency
+  # dropdown filters according to selected region
+  # method implemented in https://stackoverflow.com/questions/45975959/create-reactive-selectinput-flexdashboard-with-shiny
+  output$constituency_dropdown <- renderUI({
+    selectizeInput(inputId = "input_dropdowntext",
+                   label = "請選擇或輸入選區 / Please type or select a constituency",
+                   choices = react_region_dropdown()$DropDownText,
+                   selected = "A01: 中環 / CHUNG WAN")
```

Comment:
The warnings are quite annoying, especially when they show up in the app. Could you try do it like this example below? Might help get rid of the warnings.
http://www.baoruidata.com/examples/024-optgroup-selectize/

Reply:
Also, could this `renderUI` part go into the `# Tab: Overview of a DC` section? It's currently misplaced in the `# ----- REACTIVES ----- #` section.

Reply:
Own example here:
https://github.com/Hong-Kong-Districts-Info/dashboard-hkdistrictcouncillors/pull/70/commits/331cf99e2122be9806204e68e1dbc38ff04b34dc

Reply:
Great fix! This has worked. Also moved the `renderUI` to the appropriate section.